pkg/util/log/wrappers.go

```diff
@@ -51,3 +51,14 @@ func WithContext(ctx context.Context, l kitlog.Logger) kitlog.Logger {
 func WithSourceIPs(sourceIPs string, l log.Logger) log.Logger {
 	return log.With(l, "sourceIPs", sourceIPs)
 }
+
+// HeadersFromContext enables the logging of specified HTTP Headers that have been added to a context
+func HeadersFromContext(ctx context.Context, l log.Logger) log.Logger {
+	testMap, ok := ctx.Value(HeaderMapContextKey).(map[string]string)
```

Comment:
I probably wouldn't call this a `testMap` since we are not testing anything here.